.env.schema

```diff
@@ -4,3 +4,4 @@ REACT_APP_EGO_CLIENT_ID=ego
 # debug namespace, e.g. "app". See https://www.npmjs.com/package/debug
 REACT_APP_DEBUG=app
 REACT_APP_KEYCLOAK_ENABLED=false
+REACT_APP_PASSPORT_ENABLED=false
```

Comment:
schema file defines the basic configuration variables needed to run this app, mentioned in .readme file